File: count-the-number-of-consistent-strings/count-the-number-of-consistent-strings.py
class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        allow = set(allowed)
        cnt = 0 
        for i in words:
            for char in i:
                if char not in allow:
                    cnt += 1
                    break
        return len(words)- cnt

        

File: count-the-number-of-consistent-strings/count-the-number-of-consistent-strings.cpp
class Solution {
public:
    int countConsistentStrings(string allow, vector<string>& words) {
        
        map<char , int > mp;
        int cnt = 0 , c = 0 ;
        for(auto i : allow){
            mp[i]++;
        }
        for(auto word : words){
            for(auto ch : word){
                if(mp.find(ch)==mp.end())
                {
                    c =1 ;
                    break;
                }
            }
            if(c!=1){
                cnt++;
            }
            c = 0 ;
        }
        return cnt;


    }
};